### PR TITLE
Remote audio stream event handling for server based conference rooms

### PR DESCRIPTION
**Pull request by phanikavi**

### CODE REVIEW

File: frontend/src/Room.vue [hunk 1]
```diff
@@ -19,6 +19,7 @@
         <svg-icon v-else type="mdi" :path="mdiMicrophone"></svg-icon>
       </button>
     </div>
+    <audio id="audio" ref="audio" hidden autoplay></audio>
```
Comment (rogeru) [inline frontend/src/Room.vue:22]: There is no hidden attribute on the audio element

File: frontend/src/Room.vue [hunk 2]
```diff
@@ -238,6 +239,11 @@ export default {
         el && (el.srcObject = p.mediaStream);
       });
 
+      this.room.on('remoteAudioStream', stream => {
+        const el = this.$refs.room.querySelector(`#audio`);
+        el && (el.srcObject = stream);
```
Comment (rogeru) [inline frontend/src/Room.vue:244]: Use regular single quotes

File: frontend/src/store.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ const store = createStore({
   actions: {
     async createRoom({ state, commit }, name) {
       const isBasicAuth = state.config.AUTH_TYPE === 'BASIC_AUTH';
-      const conferenceType = state.conferenceRoomType || 'MESH';
+      const conferenceType = state.conferenceRoomType?.startsWith('SFU') ? 'SFU' : 'MESH';
```
Comment (rogeru) [inline frontend/src/store.js:49]: What is the reason for this change?

File: frontend/src/Room.vue (hunk 1 above)
Comment (phanikavi) [reply frontend/src/Room.vue]: removed 'hidden'

File: frontend/src/Room.vue (hunk 2 above)
Comment (phanikavi) [reply frontend/src/Room.vue:244]: done

File: frontend/src/store.js (hunk 3 above)
Comment (phanikavi) [reply frontend/src/store.js:49]: It was directly sending the  'Conference Room Type' values in Preferences to the backend while creating the room.  After adding text 'MESH' => 'MESH (E2E encrypted conference)' and similarly for SFU, it was breaking the room creation message.

Comment (rogeru) [reply frontend/src/store.js:49]: I see. In that case we don't handle the dropdown correct. It's my fault. I should not have changed the value name, but  instead use labels in the dropdown. I can fix that tomorrow if you like.
